src/goom_config.h

```diff
@@ -2,10 +2,10 @@
 #define GOOM_CONFIG
 
 #ifdef MID
-#define GOOM_TEXTURE_WIDTH  256
+#define GOOM_TEXTURE_WIDTH  455
```

Comment:
my concern is igpu's and embedded devices which do not support non-power-of-two textures. they may not be very common these days, i don't really follow that part of the world, but they used to be. i know for sure that those MID devices did not. most of them will have efficient padding though, i.e. embedding the texture in the smallest possible overshoot.